src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -13,6 +13,7 @@
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.codeu.data.Datastore;
 import com.google.codeu.data.User;
+import com.google.codeu.data.RegexExample;
```

Comment:
Style note: import statements are supposed to be alphabetical

Reply:
fixed that too!